plugin/pom.xml

```diff
@@ -46,6 +46,22 @@
   </developers>
 
   <dependencies>
+    <dependency>
+      <groupId>org.everit.json</groupId>
+      <artifactId>org.everit.json.schema</artifactId>
```

Comment:
Is this used?

Reply:
Yeah the JSON Schema uses it to build JSON Objects

Reply:
Where?